### ISSUE TITLE
Setting width or height of a Box does not respect isCentered property

### ISSUE THREAD

**Issue by joeynenni** [Prozi/detect-collisions#70]
If you create a Box with `isCentered: true` option and then adjust the `width` or `height` the Box is repositions itself as if `isCentered` was `false`.

In order to preserve the `isCentered` property of the box you currently need to do the following:
```js
const box = new Box({ x: 10, y: 10 }, 5, 5, { isCentered: true })

box.width = 8
box.height = 8

// extra hack to preserve isCentered property
const centroid = box.getCentroidWithoutRotation()
const x = centroid.x * (box.isCentered ? 1 : -1)
const y = centroid.y * (box.isCentered ? 1 : -1)
box.translate(-x, -y)
```

**Comment by Prozi** [comment]
@joeynenni thank you very much for this bug report

I will check this and fix in upcoming days

thank you <3

**Comment by Prozi** [comment]
fixed in `^9.4.1` thanks

**Comment by joeynenni** [comment]
I tested the fix and it works great, thank you!
